tables/sr-cyr.tbl

```diff
@@ -0,0 +1,21 @@
+#-index-name: Serbian-Cyrillic
+#-display-name: Serbian Cyrillic braille
+
+#+language:sr
+#+region:sr-cyr
```

Comment:
The correct [RFC 5646](https://tools.ietf.org/html/rfc5646) tag for `#+language` is `sr-Cyrl`.

`#+region` can be dropped.